### ISSUE TITLE
[Bug]: Custom logo throws 500 error

### ISSUE THREAD

**Issue by mffap** [zitadel/typescript#183]
### Preflight Checklist

- [X] I could not find a solution in the documentation, the existing issues or discussions
- [ ] I have joined the [ZITADEL chat](https://zitadel.com/chat)

### Describe the problem caused by this bug

Error: Invalid src prop (https://b2b-demo-rbxajm.zitadel.cloud/assets/v1/170084137439330561/policy/label/logo-dark-189254589529194753) on `next/image`, hostname "b2b-demo-rbxajm.zitadel.cloud" is not configured under images in your `next.config.js`
See more info: https://nextjs.org/docs/messages/next-image-unconfigured-host

### To reproduce

I have a custom logo configured. This error appears when calling up http://localhost:3000 in dev mode.

**Comment by mu-irfan** [comment]
Hey, gonna fix it, can you assign it to me

**Comment by hifabienne** [comment]
@mu-irfan thanks for taking this over, I assigned you to the issue.

@mffap quick question, did you set the trusted domain on your instance? 
I didn't have a look on my test setup what the exact error was, but at the beginning I also couldn't load the logo, I tested with the deploy to vercel, and either the problem was that my env var wasn't set correctly or the trusted domain.
https://zitadel.com/docs/apis/resources/admin/admin-service-add-instance-trusted-domain

**Comment by mffap** [comment]
> @mu-irfan thanks for taking this over, I assigned you to the issue.
> 
> @mffap quick question, did you set the trusted domain on your instance? I didn't have a look on my test setup what the exact error was, but at the beginning I also couldn't load the logo, I tested with the deploy to vercel, and either the problem was that my env var wasn't set correctly or the trusted domain. https://zitadel.com/docs/apis/resources/admin/admin-service-add-instance-trusted-domain

What's the value for the trusted domain? Is that supposed to be http://localhost:3000/ ?
I followed the steps [Run Login UI](https://github.com/zitadel/typescript?tab=readme-ov-file#run-login-ui), I don't think it states something about Trusted Domains.

**Comment by hifabienne** [comment]
You are right in the readme thats not stated, only in the ZITADEL docs: https://zitadel.com/docs/guides/integrate/login-ui/typescript-repo#deploy-to-vercel

The domain of the login needs to be added, so it can be used as public host from ZITADEL.
